shared/fs/filepreview/default-view.js

```diff
@@ -35,6 +35,11 @@ const DefaultView = (props: DefaultViewProps) => (
         startWithLastModified={true}
       />
     )}
+    {props.pathItem.type === 'symlink' && (
+      <Text type="BodySmall" style={stylesSymlink}>
+        {'This is a symlink' + (props.pathItem.linkTarget ? ` to: ${props.pathItem.linkTarget}.` : '.')}
```

Comment:
Since `linkTarget` isn't currently plumbed, I'd prefer if the text excluded it for now.

Reply:
You mean remove the `props.pathItem.linkTarget` test?

Reply:
Oh lol nvmind, you have it right. I just didn't read it carefully enough.